cartography/intel/gcp/__init__.py

```diff
@@ -22,14 +22,14 @@
 logger = logging.getLogger(__name__)
 Resources = namedtuple(
     'Resources', 'compute gke cloudfunction crm_v1 crm_v2 dns storage serviceusage \
-        iam admin cloudfunction apigateway cloudkms cloudrun sql bigtable firestore',
+        iam admin apigateway cloudkms cloudrun sql bigtable firestore',
```

Comment:
@sagarnavale @ajinkya1986 can you check for this file code?
i saw most of the code is missing here, for gcp please update the `cloudanix-all` branch with `cloudanix-gcp` branch